experimental/kubernetes/simple/README.md

```diff
@@ -211,3 +211,14 @@ mechanism (Redis), run
 ```
 $ bash scripts/cleanup-spinnaker.sh # This deletes everything with application name spin
 ```
+
+
+
+## Common falls
+
+1. clouddriver not configured properly, check all the logs from `clouddriver` pod, many relate to [this link](http://www.spinnaker.io/docs/troubleshooting-guide#section-my-gke-google-container-engine-cluster-isn-t-showing-up-as-a-kubernetes-account-)
+2. http://localhost:9000 hits `Access-Control-Allow-Origin` issue. Include following block to `spinnaker-local.yml`
```

Comment:
Better descriptions of symptoms would also be helpful here - e.g., "If Spinnaker's UI isn't working properly, and if your browser console has `Access-Control-Allow-Origin` errors, try including the following block in your `spinnaker-local.yml`..."